docs/man/rpm.8.scd

```diff
@@ -8,23 +8,23 @@ rpm - RPM Package Manager
 
 ## QUERYING AND VERIFYING PACKAGES
 
-*rpm* {*-q|--query*} [*select-options*] [*query-options*]
+*rpm* {*-q*|*--query*} [select-options] [query-options] [_PACKAGE_FILE_ ...] [_PACKAGE_NAME_ ...]
 
-*rpm* {*-V|--verify*} [*select-options*] [*verify-options*]
+*rpm* {*-V*|*--verify*} [select-options] [verify-options] [_PACKAGE_NAME_ ...]
 
 ## INSTALLING, UPGRADING, AND REMOVING PACKAGES
 
-*rpm* {*-i|--install*} [*install-options*] *PACKAGE_FILE ...*
+*rpm* {*-i*|*--install*} [install-options] _PACKAGE_FILE_ [...]
```

Comment:
I'm thinking that the square brackets around the ellipsis are redundant, since *that* is what the ellipsis itself means ("preceding argument can be repeated 0 or more times") :sweat_smile: 

That also seems in line with `man-pages(7)`:

> Brackets ([]) surround optional arguments, vertical  bars  (|)  separate choices, and ellipses (...) can be repeated.

Reply:
But, as soon as you start doing something like that it just feels so bloody wrong... ended up fixing it all up afterall :laughing: